packages/pigeon/platform_tests/alternate_language_test_plugin/example/macos/RunnerTests/RunnerTests.swift

```diff
@@ -0,0 +1,11 @@
+import FlutterMacOS
```

Comment:
Needs a license header I think

Reply:
I checked locally, but had some noise from generated files and missed it :(